test/Traveler-test.js

```diff
@@ -168,8 +173,9 @@ describe('Traveler', () => {
   });
 
   it('should return the total amount a traveler has spent over past year', () => {
-    traveler1.getPastTrips(tripsData);
-    traveler1.totalAmountSpent(tripsData, destinationsData);
-
+    traveler1.getPastTrips(trips);
+    traveler1.getCurrentTrip(trips);
+    let totalSpent = traveler1.totalAmountSpent(destinationsData);
+    expect(totalSpent).to.equal(12835)
```

Comment:
Everything is looking pretty solid.
I also I noticed that you are testing for `traveler1` twice, did you mean to test for a second traveler ??